projects/T-beam/include/gps.h

```diff
@@ -4,8 +4,21 @@
 #define GPS_TX_PIN 12
```

Comment:
don't use #define

Reply:
IMO the usage of defines should be addressed in a seperate pull request, given that there are a few of them in the codebase

Reply:
Don't defer work that can be done now for later, it's true that those other defines need to be addressed but this one can, and should be, changed now